locals.js

```diff
@@ -313,7 +313,7 @@ I18N["zh-CN"]["title"] = { // 标题翻译
         "Manage two-factor authentication": "管理双因素身份验证",
         "Options": "仓库 · 选项",
         "Confirm access": "授权访问",
-        "General": "通常",
+        "General": "通用",
```

Comment:
```suggestion
        "General": "常规",
```
事实上`常规`可能更合适，GitHub Mobile App 就是翻译为`常规`